### ISSUE TITLE
Allow pen buttons work without contact

### ISSUE THREAD

**Issue by hellmrf** [xournalpp/xournalpp#2664]
I was using the latest snap version of Xournal++, and in that version, I can set my stylus button for, say, the Hand tool, and then I just press the button and move the page without contact. 

However, I've noticed that, in the nightly build, this behavior was removed. Now, I need to press the button and touch the tablet at the same time to move the page. It often causes accidentals stains when taking the pen from the tablet.

It would be interesting if there was an option for changing this behavior.

Thank you!

**Comment by atx** [comment]
I agree, a configuration option for this is a good idea. My tablet has weird issues [1] with handling a button press and a pressure event at once. This makes the "hold button and use touch to erase" mode practically unusable. 

[1] For instance, holding the first button and touching down on the pad starts to magically generate events for the second button. That's what I get for buying a 20$ tablet from AliExpress I guess...

**Comment by ngoonee** [comment]
This looks like a dup of https://github.com/xournalpp/xournalpp/issues/1939
